class-00/lab-a/arrow-functions/app.js

```diff
@@ -108,45 +108,40 @@ const newObject = array => ({
 
 
 
-let sum = function(a, b, c, d) {
-  return a + b + c + d;
-};
+let sum = (a, b, c, d) =>`a + b + c + d`;
```

Comment:
Unfortunately this won't work! Instead of adding the numbers together and returning the sum, it will return the string 'a + b + c + d'